### DISCUSSION TITLE
i20211203 64bit image does not boot

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#682] [Fixed Issues]
see thread title

**Comment by hjfisch** [comment]
Update works , but new Image does not boot on RPI 4.

**Reply by klassikmann** [reply]
rern changed the images link in the README.md back to "Release i20211122". I assume that he is updating the "rOS - DIY rAudio" to generate new images.

**Reply by rern** [reply]
@klassikmann,

Please try again.

**Comment by hjfisch** [comment]
New Image seems to work  now.

**Reply by klassikmann** [reply]
Confirmed
